core/client/src/main/java/alluxio/client/block/BlockWorkerInfo.java

```diff
@@ -56,6 +56,13 @@ public long getCapacityBytes() {
   }
 
   /**
+   * @return the available bytes of the worker
+   */
+  public long getAvailableBytes() {
```

Comment:
There is some caveat that this may not return the actual available bytes of the worker, because the used bytes are updated only after a file commits. That means when there are writes in-progress, calling this method will return more bytes than the actual availability.

To counter this effect, you may want to reserve some space, say, a few GB.